src/components/CountryInfo.js

```diff
@@ -92,7 +92,14 @@ export class CountryInfo extends Component {
 
   render(_, { lockdowns, travelAdvice, coronaData, populationData }) {
     /** If the user is offline, and theres no response, or the response has failed */
-    if (!navigator.onLine) {
+
+    if (!lockdowns && !travelAdvice && !coronaData && navigator.onLine) {
+      return html`
+        Loading...
+      `;
+    }
+
+    if (navigator.onLine) {
```

Comment:
Could you explain what the bug was and why these changes are made? Im a bit confused

Reply:
I just checked the branch out locally, when you go offline and click a country, there is an error:

![image](https://user-images.githubusercontent.com/17054057/78340392-5207ab80-7596-11ea-9218-cccba2b824e5.png)

Reply:
See https://github.com/Code-for-All/lockdown/pull/115/commits/0d71bedfa66c7b587ebdf73dd9b949a66684456c

This should show the appropriate success, error, and offline states

Reply:
The Api's were offline when I was fixing. They are online now. I will update the code

Reply:
The bug was in the populationService.js.  the return statement wasn't in the right place.